backend/users/.dependency-cruiser.js

```diff
@@ -0,0 +1,3 @@
+const commonDependencyCruiserConfig = require('../../commonConfiguration/dependency-cruiser.config');
+
+module.exports = commonDependencyCruiserConfig();
```

Comment:
There is a new line missing.
This happens on several of the config files

Reply:
Thanks @adriencaccia we fixed this ;)